### PR TITLE
latest changes merging from release-3.4.0 to release-3.5.0

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Review by amit-tarento — APPROVED** [review]

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java [hunk 1]
```diff
@@ -259,6 +261,7 @@ private void updateUser(Request actorMessage) {
       isPrivate = (boolean) actorMessage.getContext().get(JsonKey.PRIVATE);
     }
     Map<String, Object> userMap = actorMessage.getRequest();
+    logger.info(actorMessage.getRequestContext(), "Incoming update request body: " + userMap);
```
Comment (amit-tarento) [inline actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java:264]: remove this log

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java [hunk 2]
```diff
@@ -321,18 +322,21 @@ private void updateUser(Request actorMessage) {
...
     boolean resetPasswordLink = false;
     if (StringUtils.isNotEmpty(managedById)
-            && (StringUtils.isNotEmpty((String) requestMap.get(JsonKey.EMAIL)))
-        || (StringUtils.isNotEmpty((String) requestMap.get(JsonKey.PHONE)))) {
+        && ((StringUtils.isNotEmpty((String) requestMap.get(JsonKey.EMAIL))
+            || (StringUtils.isNotEmpty((String) requestMap.get(JsonKey.PHONE)))))) {
       requestMap.put(JsonKey.MANAGED_BY, null);
       resetPasswordLink = true;
     }
+    logger.info(actorMessage.getRequestContext(), "Update request body for user db: " + requestMap);
```
Comment (amit-tarento) [inline actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java:330]: remove this log

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java [hunk 3]
```diff
@@ -46,6 +46,21 @@ public Response insertRecord(
       String keyspaceName, String tableName, Map<String, Object> request, RequestContext context) {
     long startTime = System.currentTimeMillis();
     logger.info(context, "Cassandra Service insertRecord method started at ==" + startTime);
+    if (tableName.equalsIgnoreCase(JsonKey.USER)) {
```
Comment (amit-tarento) [inline actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java:49]: remove this log

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java [hunk 4]
```diff
@@ -90,6 +105,22 @@ public Response updateRecord(
       String keyspaceName, String tableName, Map<String, Object> request, RequestContext context) {
     long startTime = System.currentTimeMillis();
     logger.info(context, "Cassandra Service updateRecord method started at ==" + startTime);
+    if (tableName.equalsIgnoreCase(JsonKey.USER)) {
+      try {
```
Comment (amit-tarento) [inline actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java:109]: remove this log

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java [hunk 5]
```diff
@@ -377,11 +409,25 @@ public Response getAllRecords(
     return response;
   }
 
-  @Override
   public Response upsertRecord(
       String keyspaceName, String tableName, Map<String, Object> request, RequestContext context) {
     long startTime = System.currentTimeMillis();
     logger.info(context, "Cassandra Service upsertRecord method started at ==" + startTime);
+    if (tableName.equalsIgnoreCase(JsonKey.USER)) {
```
Comment (amit-tarento) [inline actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java:416]: remove this log

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java (hunk 2 above)
Comment (Hari-stackroute) [reply actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java]: Resolved.

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java (hunk 3 above)
Comment (Hari-stackroute) [reply actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java]: Resolved.

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java (hunk 4 above)
Comment (Hari-stackroute) [reply actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java]: Resolved

File: actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java (hunk 5 above)
Comment (Hari-stackroute) [reply actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-cassandra-utils/src/main/java/org/sunbird/cassandraimpl/CassandraOperationImpl.java]: Resolved.

File: actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java (hunk 1 above)
Comment (Hari-stackroute) [reply actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java:264]: As of now we kept only this log.

**Pull request merged**